### PR TITLE
[!!!][TASK] Replace grunt with gulp

### PR DESCRIPTION
**Pull request by xf-**
## Description

Replace grunt with gulp

## Motivation and Context

- Grunt is more config based and gulp code based. We need less/no hacky solutions.
- It is maybe faster.
- Modules we need are up2date.
- Small changes can be done via NodeJS without a plugin

## How Has This Been Tested?

Run the tasks

## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     {
       "name": "Freifunk Regensburg",
       // Please ask Freifunk Regensburg before using its tile server c- example with retina tiles
-      "url": "https://{s}.tiles.ffrgb.net/{z}/{x}/{y}{retina}.png",
+      "url": "https://1.tiles.ffrgb.net/{z}/{x}/{y}{retina}.png",
```
Comment (genofire) [inline config.json]: bad -> maybe `"subdomains":"1"`, ?

**Review by genofire — CHANGES_REQUESTED** [review]
I looks good -> or is something not implemented yet?

Do you change the subdomains on config?

File: config.json (hunk 1 above)
Comment (xf-) [reply config.json]: I will remove subdomains - we use http2 and no need for multiple dns

Comment (genofire) [reply config.json]: Then remove `subdomains` entry

Comment (xf-) [reply config.json]: yeah

**Review by genofire — APPROVED** [review]

**Pull request merged**